components/Table/TableContainer.jsx

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
+import HardwareKeyboardArrowRight from 'material-ui/svg-icons/hardware/keyboard-arrow-right';
 import styles from './TableContainer.css';
 
 const TableContainer = ({ title, style, children }) => (
   <div className={styles.container} style={{ ...style }}>
     <div className={styles.heroesContainer}>
-      <div className={styles.tableHeading}>{title}</div>
+      <div className={styles.tableHeading}>
+        <HardwareKeyboardArrowRight style={{ verticalAlign: 'top', opacity: '.6', height: 26 }} />
+        {title}
+      </div>
```

Comment:
Do we plan on making the table name clickable in some way? Otherwise, it feels like it might trick users into thinking they can since that kind of arrow typically signifies that there is more info or options if the item is clicked.